spec/current_order_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'current_order'
+require_relative 'takeaway_spec'
+
+describe CurrentOrder do
+
+  let(:menu_item) { double }
+
+  describe '#add_item' do
+    it 'adds an item to the current oder' do
+      subject.add_item(menu_item)
+      expect(subject.order).to eq([menu_item])
+    end
+  end
+
+  describe '#total' do
```

Comment:
I was a bit confused by this because `TEST_ORDER` is defined in another file.  I suspect your intention is to avoid duplication, which is good, but the cost is indirection and increased cognitive load.